### DISCUSSION TITLE
2023/07/history/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#86] [General]
# 2023/07/history/

https://xiangyun.rbind.io/2023/07/history/

**Comment by earfanfan** [comment]
“历史是任人打扮的小姑娘”这句里面的“历史”应该是说后人述说的前人的历史，或者指众多流派中的一种历史观，而不是指真实的历史。同理，你所追求的客观和全面，也不可能是一种真正的客观和全面认知世界的方式，也只会是弱水三千只取一瓢中的一瓢，以人类的有限，可能根本就不存在绝对的、确定性的“客观和全面”。从现在的我的角度看，如果你非要追求绝对的客观和全面，以后只会不断碰壁，是在走一条没有终点的路，极有可能会越走越狭隘。当然啦，如果是我理解错了，你本来追求的就是相对的客观和全面，那么随着量的累积，终会有由量变达成质变的那天的。

确实，时至今日人们脑子里还是存在太多封建时代遗留下来的思想。不知道那位老师在提出这个问题后，还有没有继续深入地分析一下这个问题是怎么形成的，在未来的未来这个问题又会怎样变化。

我最近硬着头皮啃一本社会学的书，从社会学的观点看，人们脑子里的思想是和传统文化、社会制度共生的。这里举几个例子。第一个例子，思想是和文化共生的，在一些经济上落后的农村地区存在着“女人不能上桌吃饭、不能祭祀祖宗的习俗”（ps注意这里我不是想打拳，是举例子），在遵从这个习俗的人的脑子里，这个习俗不仅仅只是一种思想，更加是一个不能推翻的准则，而这个准则和在村子里如何播种种地一样，是一齐流传下来的，正因如此，没有办法做到只是将人们脑子里那些被现在认为是糟粕的思想剥离开并且推翻，却丝毫不改那些被现在认为不是糟粕的思想。

第二个例子，思想是和社会制度共生的，我们国家现在是社会主义国家，但是我们的政府仍然是集权式、家长式的政府，政治制度有了大变化，但是社会制度还是这样，那是因为维护社会制度的人还没那么快改变。在社会变迁的过程中，人们的需求是在变化的，所以人民思想的变化最先就发生在人民身边，但是领导阶层却似乎总是不能那么快地适应环境的变动，这种错位……对于“社会制度”我的理解也还是很模糊不清的，这里干脆举一个我认识更多一点的具体例子。以我们国家的户籍政策为例，在明代的时候，户籍政策将人与职业、土地固定，农民的后代世代为农、铁匠的后代世代打铁，户籍制度控制了人的流动也将税源固定。而现在，户籍政策还是亟待改革，比如在城市里，户籍能绑定牵扯的利益资源太多了。即便人们对于这些牵涉生活细节的社会制度的想法可以改变，但是如果这社会制度不变，人们为了更好地适应生活，也还是要继续被从遵从它，毕竟不会有人非得跟自己的生活过不去。当然啦，我写到这里，并不是想把锅全扣到领导阶层的头上，毕竟社会分成了很多层，其中一个层想要做出改变，还有很多层不想要改变，所有层的利益不会是一致的。

最后你提到流浪地球，其实你并没有和大众距离越来越远啊，大众并没有你想象的那么那么……我看到那些情节时也没撒情绪波动，因为看的时候感觉套路太明显了，基本能猜到后面会发生撒剧情。另外，想让我们哭的未必就是你想象中的政治宣传因素，也可能就是电影主创团队想制造泪点和记忆点，便于营销和传播。除了政治权力，还有资本逐利，这几年主旋律电影票房大卖，就算没有权力控制，资本看到这么好赚钱也会自发继续拍的。

**Reply by XiangyunHuang** [reply]
> 从现在的我的角度看，如果你非要追求绝对的客观和全面，以后只会不断碰壁，是在走一条没有终点的路，极有可能会越走越狭隘。当然啦，如果是我理解错了，你本来追求的就是相对的客观和全面，那么随着量的累积，终会有由量变达成质变的那天的。

诚如你所说，绝对客观和全面是不存在的，即使我去刷一些历史课，那些历史学家也说是根据仅有的公开的材料来叙述过去，不可能有绝对客观和全面哈。我这里强调的相对，相对的是那些千篇一律的、只有一个解释的、而且距离历史事实有相当遥远距离的历史正剧、电影等作品。

> 确实，时至今日人们脑子里还是存在太多封建时代遗留下来的思想。不知道那位老师在提出这个问题后，还有没有继续深入地分析一下这个问题是怎么形成的，在未来的未来这个问题又会怎样变化。

从高华老师历史课中的历史事实来看，是通过树立一些正面、批判一些反面以及经过精心设计的历史叙述，再通过各种媒体长期地宣传。比如宣扬历史上的圣人、清官（海瑞、包拯等），批判贪官（严嵩、和珅等），宣扬忠义仁孝的典故（关云长、诸葛亮）等等，让百姓期待有一个清官，再通过清官来爱护百姓。而不是让百姓觉醒获得监督政府的民主权利，监督政府官员兢兢业业为人民服务，否则百姓就可以弹劾政府官员。天地君亲师外加三纲五常，君权天授，这也是有一套精心的、小心翼翼地构建的叙述。

从来都是自上而下统治，而没有自下而上的监督。在这方面，我对未来不抱希望，没有任何变化的可能。

> 女人不能上桌吃饭、不能祭祀祖宗的习俗。

毫无疑问，这就是封建思想呀，《菊豆》这种反封建的作品不好找的。世俗的力量是非常强大的，现在不谈反封建是有原因的。世俗力量本身也可以看作一种统治社会的力量，统治阶级不可能将自己手中可用的统治力量主动削弱。女子缠足，慈禧一句话，就可以破除了。男子剃发、孙中山一句话，就剃了，这种外在的很好做。但是，思想的转变需要很多代人慢慢地化解，跟生产力水平也有关系。不过想想，现在和100年前还是有很多变化的。

> 最后你提到流浪地球，.... ，另外，想让我们哭的未必就是你想象中的政治宣传因素，也可能就是电影主创团队想制造泪点和记忆点，便于营销和传播。

也许是我有点过敏和警惕，感觉这种主旋律越来越多，而且一出来就占据舆论焦点，我觉得是在营造一种社会气氛。

>  除了政治权力，还有资本逐利，这几年主旋律电影票房大卖，就算没有权力控制，资本看到这么好赚钱也会自发继续拍的。

一部分导演是这样想的，特别是背后那些投资人。导演有不少还是有一个梦的，想通过电影去反应生活、艺术、给人启发、思考的，而不仅仅是迎合制造出来主旋律。也有很多导演不缺钱，真要缺钱也不是一般意义上的缺钱，也有为了完成一部电影而缺钱的，就是为了做一件自己认为很有意义的事而缺钱。这种类型的电影，拍摄难度高、演员要求也很高、拍摄内容限制又多，最后，不少导演会被迫走上拍摄商业和主旋律片的。

**Reply by XiangyunHuang** [reply]
《菊豆》巩俐主演的，张艺谋导演的，给人感觉很压抑，十足的文艺片。《1942》也是一部很好的写实的片子，冯小刚导演的，张国立主演的。

**Reply by XiangyunHuang** [reply]
我想我还得树立一个就在眼前的「只讲一面、不讲另一面」的反面典型，来形象地表达我所追求的客观和全面，比如这个视频[ChatGPT王炸级更新！Code Interpreter 10大应用，数据分析师会失业吗？](https://youtu.be/0rOFnC51mx0)。里面讲了很多 ChatGPT 的功能，并根据这些功能下结论说可以取代一堆岗位。但，实际上，只要真的是做过商分的就知道，ChatGPT 只是个工具，数据分析师还要做很多工具不能解决的问题，分析思路、业务模型、团队沟通等等，而这些方面耗费的时间远大于去做数据分析，编程分析/展示只是最后一步，换个人也能做，换个新工具做得更快。新的工具可以在用到工具的时候提效，比如完全不会编程的人，还可以借助它来做粗浅的描述统计。随着 ChatGPT 等新工具的出现，类似的贩卖焦虑的标题党越来越多，过度泛滥必将引火烧身。

**Reply by earfanfan** [reply]
你这让我越发不懂你所追求的客观和全面是撒了，反而开始反思是不是不该吐槽你，好像弄得你有点纠结。其实，从我的角度看，你说“唤醒民众”的三步本来也是一个大槽点，不过我昨天拿不准我俩可不可以交流这部分的分歧，就没写了。

**Reply by XiangyunHuang** [reply]
ChatGPT 乱入而已，我也觉得有些不妥。

**Reply by earfanfan** [reply]
哈哈，是啊，咱俩原来讨论的“全面和客观”不是认知历史、历史观方面的嘛，跟数据分析、chatGPT 相比，前者更务虚，后者偏向于工具和方法也更务实。

**Reply by XiangyunHuang** [reply]
追求客观和全面的态度不应该局限于对待历史，现在，自媒体时代，为了搏流量，标题党，观点、内容走极端例子很多，上面引述的 chatGPT 又是一例而已。

**Reply by botsp** [reply]
所谓历史的真实，好像史学研究方法学上有个"第一历史、第二历史"的说法。如果没有绝对的真实，那自然给各种"不真实"留下了演绎的空间，但这并不应该严重到历史任人摆弄。我们起码是追求真实的，出现[历史是任人打扮的小姑娘]这种现象，实在是文明的耻辱了。

就挑近的说，当权者动不动就要把历史的"解释权"攥在自己手里，对于不符合自身利益的事实，就绝不承认、自己去随意打扮歪曲。别人稍微讲几句实话，就扣一顶"历史虚无主义"的帽子。看似只是在给真实下定义，其实是强奸全体民众的智商。
@earfanfan/@xiangyunhuang   拼音名字就是好，湘云、湘云，哈哈

**Reply by XiangyunHuang** [reply]
拥有解释权，再通过大量的媒体进行个性化的推送，十年如一日，产生的教育力量是不可小觑的。我想还是要多看看，看不一样的东西，100 年前蔡元培的「思想自由、兼容并包」，陈寅恪的「独立之精神，自由之思想」，梅贻琦的「所谓大学者，非谓有大楼之谓也，有大师之谓也」等等，放在现在仍然振聋发聩。

记国博「百年巨匠」展览

**Reply by botsp** [reply]
"个性化推送" 这真是妙，技术工具给公权力登峰造极了。

**Comment by XiangyunHuang** [comment]
意识形态领域没有一刻真空过。今天发现一个令人骇然的事，《小 Lin 说》栏目更新了一期关于中国房地产市场的视频，介绍最近发生的一系列事件，整理、串联、分析了不少材料。上架不到半天，很快在 B 站、油管下架。令我感到骇然的点是之前听说 XX 被封杀都只是听说，这下是在我的关注圈内发生的，而且我目睹了这个过程。B 站剩下的那些可播的视频都是些什么视频，强烈的对比，让人感到窒息。之前，我一直好奇 《小 Lin 说》栏目讲了那么多国外的情况，为什么没有一期是讲中国国内的事件。我既好奇又期待，我不相信作为一个中国人，她会忍住不讲。今天她讲了，她很勇敢。《小 Lin 说》栏目的影响力那么大，不知道后续会怎么样？

**Reply by XiangyunHuang** [reply]
刘邦说「嗟呼，大丈夫当如是也！」，项羽说「彼可取而代之。」秦始皇在刘邦、项羽身旁经过，刘邦说我要成为秦始皇那样的人，他想推翻的就是秦始皇这个人而已，取而代之。几千年下来，这种思想一直都在，他们一面反对体制化，一面又想做体制化的受益者，等成为受益者之后，加固体制化，让别人不要上来了。最近 100 多年，人类除了科技在进步，人还是那些人，甚至退步了。最近几年，伴随着一点点不断增进对社会的了解越来越迷茫，真了解了真相之后，是好还是坏呢？罗曼·罗兰说「世上只有一种英雄主义，就是在认清生活真相之后依然热爱生活」，鲁迅说「真的猛士，敢于直面惨淡的人生，敢于正视淋漓的鲜血。」当下的生活真的需要点鸡血和鸡汤，自我洗脑。

记于 2023-09-10 。

**Reply by botsp** [reply]
同意。刘邦项羽就不讲了，单就社会习俗来说，他们不是反对压迫、也不是向往自由、平等的权利，所谓的抱怨只是渴望权力、所谓的反抗也只是想要获得权力而已。

我们需要真正的变革，勇敢追求真正的价值。

另外，我觉得不应该推之到人类的范围，目所能及的，只有长江黄河水是这样子。先进文明几乎不浪费历史的机遇，永远在前进，很少原地打转。
